priv/eleveldb.schema

```diff
@@ -239,3 +239,31 @@
   {datatype, {enum, [true, false]}},
   hidden
 ]}.
+
+%% @doc Enable global expiry.  All leveldb databases / vnodes
+%% will use same expiry_minutes and whole_file_expiry settings.
+%% Same settings apply to all leveldb instances in multi_backend.
+{mapping, "leveldb.expiry_enabled", "eleveldb.expiry_enabled", [
+  {default, off},
+  {datatype, flag},
+  hidden
+]}.
+
+%% @doc Minutes until object expires.  Gives number of minutes
+%% a stored key/value will stay within the database before automatic
+%% deletion.  Zero disables expiry by age (minutes) feature
+{mapping, "leveldb.expiry_minutes", "eleveldb.expiry_minutes", [
+  {default, 0},
+  {datatype, integer},
```

Comment:
cuttlefish is its own little world.  I do copy/paste of previous options as my base.  The enif_get_XxX() functions used in eleveldb.cc are defined as filtering out of range values.  Therefore my defense is at that layer, not here or other weirdness in eleveldb.erl.